docs/reference/tab-widgets/troubleshooting/data/migrate-to-data-tiers-routing-guide.asciidoc

```diff
@@ -0,0 +1,224 @@
+// tag::cloud[]
+In order to get the shards assigned we'll need to call the 
+<<ilm-migrate-to-data-tiers, migrate to data tiers routing>> API which will
+resolve the conflicting routing configurations towards using the standardised
+<<data-tiers, data tiers>>. This will also future-proof the system by migrating 
+the index templates and ILM policies if needed.
+
+**Use {kib}**
+
+//tag::kibana-api-ex[]
+. Log in to the {ess-console}[{ecloud} console].
++
+
+. On the **Elasticsearch Service** panel, click the name of your deployment. 
++
+
+NOTE:
+If the name of your deployment is disabled your {kib} instances might be
```

Comment:
The "the name of your deployment" is disabled? Is that a type, or can the name of something be disabled?

Reply:
Maybe we just change this to "If your deployment is disabled" instead? It reads just a little strange to me. Might just be me though.